xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/DXVAHD.cpp

```diff
@@ -669,8 +694,6 @@ bool CProcessorHD::Render(CRect src, CRect dst, ID3D11Resource* target, CRenderB
   m_pVideoContext->VideoProcessorSetStreamSourceRect(m_pVideoProcessor.Get(), DEFAULT_STREAM_INDEX, TRUE, &sourceRECT);
   // Stream dest rect
   m_pVideoContext->VideoProcessorSetStreamDestRect(m_pVideoProcessor.Get(), DEFAULT_STREAM_INDEX, TRUE, &dstRECT);
-  // Output rect
-  m_pVideoContext->VideoProcessorSetOutputTargetRect(m_pVideoProcessor.Get(), TRUE, &dstRECT);
```

Comment:
With RTX 4070 found this causes vertical shift of some pixels in windowed mode (i.e video cut in top and a black band in bottom or not updated pixels). I'm not sure exactly what it's due to or if this problem already existed before or it's from a driver update or just only RTX 40x... (not happens with Intel)

Removed because according Microsoft description not seems necessary:

>**Remarks**
 If this method is never called, or if the Enable parameter is FALSE, the video processor writes to the entire destination surface.

Reply:
resote this. it will not work correctly if video window not fill screen. like it was on PVR on Confluence theme

Reply:
OK investigated more and also occurs in full screen but ONLY when super resolution is enabled. Then used bool flag (before always TRUE) to disable only when super resolution is used. I think this workaround should be acceptable for now... Maybe is a Nvidia implementation bug and gets fixed with a driver update.

Reply:
I think there should be a comment explaining that, with the nVidia driver version tested, so it can be remembered and removed if a driver update fixes it.

Does Chrome have something like this?

Reply:
I will add a code comment tomorrow

Reply:
Does Intel have the same bug?